test/unit/browser/renderer.html

```diff
@@ -116,15 +116,22 @@
 			runner.on('pending', test => window.mocha_report('pending', serializeRunnable(test)));
 		};
 
+
+		async function loadModules(modules) {
```

Comment:
Changing how we load files was necessary for Mocha to populate the `currentTest.file`, which we need to know where to place snapshots